components/twitter-card.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import {fallbackLocale} from './locale'
+
+function TwitterCard({
+  card = 'summary_large_image',
+  title = 'Maintainable CSS with React',
+  description = 'React component styling solved with an elegant API, small footprint, and great performance',
+  pathname,
+  locale,
+}) {
+  const urlPrefix = fallbackLocale === locale ? '' : `${locale}.`
+  const image = `https://${urlPrefix}glamorous.rocks/static/images/banner.png`
```

Comment:
@kentcdodds @paulmolluzzo Sorry about that. I can try work on this to pick up using `req`. Basically, what I will do is to have another more `req` parameter on the `twitter-card` component and pass in the `req` value from the pages.

What would you guys prefer?